File: coding-test/java/Class_02/bj_11050.java
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.StringTokenizer;

public class bj_11050 {

    public static int factorial(int n) {
        int result = 1;
        for(int i = 1; i <= n; i++) {
            result *= i;
        }
        return result;
    }

    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        StringTokenizer st = new StringTokenizer(br.readLine());

        int N = Integer.parseInt(st.nextToken());
        int K = Integer.parseInt(st.nextToken());

        int result = factorial(N) / (factorial(K) * factorial(N - K));
        System.out.println(result);
    }
}

File: coding-test/python/Class_02/bj_11050.py
def factorial(n):
    result = 1
    for i in range(1, n+1):
        result *= i
    return result

N, K = map(int, input().split())

result = factorial(N) // (factorial(K) * factorial(N - K))
print(result)

""" 
import math
N, K = map(int, input().split())
print(math.factorial(N) // (math.factorial(K) * math.factorial(N-K)))
"""